copy_this/application/commands/cacheclearcommand.php

```diff
@@ -48,12 +48,20 @@ public function help(oxIOutput $oOutput)
     public function execute(oxIOutput $oOutput)
     {
         $oInput = $this->getInput();
+
+        $oOutput->writeLn('Clearing OXID db and backend cache..');
+        /** @var oxCache $oCache */
+        $oCache = oxNew('oxcache');
```

Comment:
I'd suggest move everything that is related to oxCache to separate `cache:flush` command